src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-param-reassign */
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: (state) => {
+      state.categories = 'Under construction';
+    },
+  },
+});
+
+export const { addCategory } = categoriesSlice.actions;
```

Comment:
You created the `checkStatus` action but exported `addCategory`.  This will lead to error, kindly reconcile the names.